docs/devel/api-change-guidelines.md

```diff
@@ -0,0 +1,21 @@
+# API Change Guidelines
+
+Freely change pkg/api/types.go. 
+
+Update the most recent versioned package's conversion functions. (In, e.g., pkg/api/v1beta1/conversion.go)
```

Comment:
Actually, all the older package's conversion functions are the ones that should be updated.